Defining "algorithmic opacity" (#252)
tatianamac/selfdefined: 11ty/definitions/algorithmic-opacity.md

---

> title: Algorithmic Opacity

Reviewer: ```suggestion
title: algorithmic opacity
```

slug: algorithmic-opacity
defined: true
speech: noun
---

> The effect of an algorithmic system designed in a way that makes difficult to explain why a certain decision was made by it.

Reviewer: ```suggestion
the effect of an algorithmic system designed in a way that makes difficult to explain why a certain decision was made by it.
```

I think here we should avoid defining it in a partially self-referential way? Can we define the "algorithmic" aspect of it to avoid this?

Author: Would "when an algorithmic system (a set of instructions, tasks, or processes to accomplish a task) is designed in a way that makes its decision making difficult to explain or understand"

Reviewer: the effect of a system (e.g., instructions, tasks, or processes to accomplish a task) that designed in a way that makes its decision making difficult to explain or understand

> An algorithmic system may be kept opaque deliberately to conceal how it operates, unintentionally due to the expert technical knowledge required to understand it, or as result of a mismatch between how humans and algorithms understand the input given.

Reviewer: ```suggestion
## Issues
An algorithmic system may be kept opaque deliberately to conceal how it operates, unintentionally due to the expert technical knowledge required to understand it, or as result of a mismatch between how humans and algorithms understand the input given.
```
Is there the potential for intentional harm done by humans by way of the algorithm?

## Impact

> When concealed, this prevents users/consumers from fully understanding what these algorithms can and cannot do. Ultimately, lack of transparency allows for biases to remain unchecked, and this creates potential for additional harm to be done.

Reviewer: ```suggestion
When concealed, algorithmic opacity prevents users/consumers from fully understanding what these algorithms can and cannot do. Ultimately, lack of transparency allows for human-led biases to remain unchecked, which creates the potential for additional harm.
```
